docs/esp8266/tutorial/neopixel.rst

```diff
@@ -14,6 +14,13 @@ To create a NeoPixel object do the following::
 This configures a NeoPixel strip on GPIO4 with 8 pixels.  You can adjust the
 "4" (pin number) and the "8" (number of pixel) to suit your set up.
 
+For LEDs with more than 3 colors, such as RGBW pixels or RGBY pixels, the 
+NeoPixel class takes a ``bpp`` parameter. To setup a NeoPixel object for an 
+RGBW Pixel, do the following::
+
+    >>> import machine, neopixel
+    >>> np = neopixel.NeoPixel(machine.Pin(4), 8, bpp=4)
+
```

Comment:
You might want to mention that when the NeoPixel object is in 4-bpp mode then all the colour tuples (eg in the demos below) need to be 4-tuples, as opposed to 3-tuples.

Reply:
I've added an example as well and reordered the sections. If it doesn't make sense I'm open to further adjustments.